ecl/hql/hqlgram.y

```diff
@@ -7976,27 +7976,29 @@ simpleDataSet
                             $$.setExpr(createDataset(no_choosesets, $3.getExpr(), $5.getExpr()));
                             $$.setPosition($1);
                         }
-    | DEDUP '(' startTopLeftRightSeqFilter optDedupFlags ')' endTopLeftRightFilter endSelectorSequence
+    | DEDUP '(' startSortOrder startTopLeftRightSeqFilter optDedupFlags ')' endTopLeftRightFilter endSelectorSequence endSortOrder
```

Comment:
Jake I don't really understand your suggestion.  What are you hoping your syntax would do?